build_scripts/start_site.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -euo pipefail
+ENV_FILE=.env
+
+if test -f "$ENV_FILE"; then
+  export $(cat .env | xargs)
+fi
+
+if [[ -z "${EXTERNAL_SITE_PATH}" ]]; then
+  # Run the site without external content
```

Comment:
I think we should error if `EXTERNAL_SITE_PATH` is not set. A developer should have both ocw-www and this repo checked out, and have the environment variables set up to point to the directory